.github/ISSUE_TEMPLATE/end_game.md

```diff
@@ -122,6 +122,7 @@ assignees: ''
 - [ ] Trigger algolia search crawler [workflow](https://github.com/spiceai/docs/actions/workflows/trigger_search_reindex.yml), to reindex updated docs.
 - [ ] Update the [Helm chart](https://github.com/spiceai/spiceai/blob/trunk/deploy/chart) version (image.tag version & chart version). Ensure [docker build](https://github.com/spiceai/spiceai/actions/workflows/spiced_docker.yml) for the tag from the release branch completed (~2 hours) and trigger the [Release Chart](https://github.com/spiceai/helm-charts/actions/workflows/release.yml) workflow.
       **Note**: Release chart workflow should be triggered only after the [binaries have finished building](https://github.com/spiceai/spiceai/actions/workflows/build_and_release.yml) and [docker image have finished building](https://github.com/spiceai/spiceai/actions/workflows/spiced_docker.yml).
+- [ ] Set the [release](https://github.com/spiceai/spiceai/releases) as latest release after the [binaries have finished building](https://github.com/spiceai/spiceai/actions/workflows/build_and_release.yml) and [docker image have finished building](https://github.com/spiceai/spiceai/actions/workflows/spiced_docker.yml)
```

Comment:
The installation script for binaries checks against the "latest" version, so we need to set the latest version before testing the E2E release installation.